packages/kovan-faucets/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@0xproject/erc20-token-faucet",
+  "version": "1.0.0",
+  "description": "A faucet micro-service that dispenses test ERC20 tokens",
+  "main": "server.js",
+  "scripts": {
+    "build": "node ../../node_modules/gulp/bin/gulp.js build",
+    "dev": "node ../../node_modules/gulp/bin/gulp.js run",
+    "start": "node ./bin/server.js",
+    "lint": "tslint src/**/*.ts",
+    "clean": "shx rm -rf bin"
+  },
+  "author": "Fabio Berger",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "0x.js": "~0.25.1",
+    "bignumber.js": "~4.1.0",
+    "body-parser": "^1.17.1",
+    "es6-promisify": "^5.0.0",
+    "ethereumjs-tx": "^1.2.5",
+    "express": "^4.15.2",
+    "lodash": "^4.17.4",
+    "rollbar": "^0.6.5",
+    "web3": "^0.20.0",
```

Comment:
Can we make sure we are using the same `web3` version between all the sub-packages?